packages/context/test/acceptance/context-view.acceptance.ts

```diff
@@ -0,0 +1,182 @@
+import {expect} from '@loopback/testlab';
+import {
+  Context,
+  ContextView,
+  ContextEventListener,
+  Getter,
+  inject,
+  ContextEventType,
+  Binding,
+} from '../..';
+
+describe('ContextView - watches matching bindings', () => {
+  let server: Context;
+  let contextWatcher: ContextView;
+  beforeEach(givenControllerWatcher);
+
+  it('watches matching bindings', async () => {
+    // We have ctx: 1, parent: 2
+    expect(await getControllers()).to.eql(['1', '2']);
+    server.unbind('controllers.1');
+    // Now we have parent: 2
+    expect(await getControllers()).to.eql(['2']);
+    server.parent!.unbind('controllers.2');
+    // All controllers are gone from the context chain
+    expect(await getControllers()).to.eql([]);
+    // Add a new controller - ctx: 3
+    givenController(server, '3');
+    expect(await getControllers()).to.eql(['3']);
+  });
+
+  function givenControllerWatcher() {
+    server = givenServerWithinAnApp();
+    contextWatcher = server.watch(Context.bindingTagFilter('controller'));
+    givenController(server, '1');
+    givenController(server.parent!, '2');
+  }
+
+  function givenController(_ctx: Context, _name: string) {
```

Comment:
Why are the argument names starting with an underscore (`_ctx` and `_name`)? Can you use just `ctx` and `name` please?